src/audio/igo_nr/igo_nr.c

```diff
@@ -43,16 +43,30 @@ DECLARE_SOF_RT_UUID("igo-nr", igo_nr_uuid,  0x696ae2bc, 0x2877, 0x11eb, 0xad, 0x
 
 DECLARE_TR_CTX(igo_nr_tr, SOF_UUID(igo_nr_uuid), LOG_LEVEL_INFO);
 
+static void igo_nr_lib_process(struct comp_data *cd)
+{
+	if (!cd->process_enable[0] || cd->invalid_param) {
```

Comment:
Can you comment on what we are checking for here ?
Btw, it looks like the commit message has been wrapped incorrectly ?

Reply:
Certainly. I will update a comment and new commit.